pkg/v1/providers/infrastructure-vsphere/v1.3.1/yttcc/overlay.yaml

```diff
@@ -43,9 +40,15 @@ spec:
   topology:
     class: #@ data.values.CLUSTER_CLASS
     #! VVV TODO(vui) compute
-    version: #@ data.values.KUBERNETES_VERSION
+    #! version: #@ data.values.KUBERNETES_VERSION
+    #! hack to match up tkr
+    version: #@ "{}-tkg.1-zshippable".format(data.values.KUBERNETES_VERSION)
     controlPlane:
       replicas: #@ data.values.CONTROL_PLANE_MACHINE_COUNT
+      #@overlay/match missing_ok=True
+      metadata:
+        annotations:
+          run.tanzu.vmware.com/resolve-os-image: image-type=ova,os-name=photon
```

Comment:
we will need to populate the `resolve-os-image` dynamically. that can be saved for a separate issue for later.